CHANGELOG.md

```diff
@@ -17,6 +19,9 @@ The public API of this library consists of the functions declared in file
     - exactEdgeLengthRads
     - exactEdgeLengthKm
     - exactEdgeLengthM
+- refactor `getH3UnidirectionalEdgeBoundary` for accuracy at small resolutions (#391)
+    - speeds up `getH3UnidirectionalEdgeBoundary` by about 3x
+    - implement core logic for future vertex mode
```

Comment:
Thanks for adding. Technically I guess this is a "fix", rather than a feature, but I think this is fine.